bundle/android/strings.xml

```diff
@@ -348,10 +354,9 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="gutenberg_native_slash_inserter_results" tools:ignore="UnusedResources">Slash inserter results</string>
     <string name="gutenberg_native_some_blocks_have_additional_settings_tap_the_settings_icon_on_the" tools:ignore="UnusedResources">Some blocks have additional settings. Tap the settings icon on the bottom right of the block to view more options.</string>
     <string name="gutenberg_native_start_writing" tools:ignore="UnusedResources">Start writing…</string>
-    <string name="gutenberg_native_take_a_photo" tools:ignore="UnusedResources">Take a Photo</string>
-    <string name="gutenberg_native_take_a_photo_or_video" tools:ignore="UnusedResources">Take a Photo or Video</string>
-    <string name="gutenberg_native_take_a_video" tools:ignore="UnusedResources">Take a Video</string>
```

Comment:
https://github.com/wordpress-mobile/gutenberg-mobile/pull/6204#pullrequestreview-1626568719, the i18n script is marking these strings as used by the web, so it considers that we don't need to translate them as they would be part of the [Gutenberg GlotPress project](https://translate.wordpress.org/locale/en-gb/default/wp-plugins/gutenberg/). However, I checked that these strings are only referenced by native files, hence they won't be translated unless we add them here.

I presume that the fact that these strings are [in an extension different from `.native.js`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/media-upload/constants.js) is making the i18n script consider them as used on the web. Looks like the constants associated with the localized strings are only used in an integration test, not sure if this might confuse the i18n script regarding whether they are used in web or not. In any case, as a workaround, we could simply move the test-only constants to a separate file under the `test` folder. I'll open a separate PR to address this and prevent this issue in future releases.

Reply:
Fixed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6243/commits/5ac4d8be84db8e6bc8431862d97f9d7e6729872a.

Reply:
Next releases won't encounter this issue once we merge https://github.com/WordPress/gutenberg/pull/54928.